fieldservice_recurring/models/fsm_recurring.py

```diff
@@ -78,10 +78,28 @@ def _default_team_id(self):
         tracking=True,
     )
     person_id = fields.Many2one(
-        "fsm.person", string="Assigned To", index=True, tracking=True
+        "fsm.person",
+        string="Assigned To",
+        compute="_compute_person_id",
+        precompute=True,
+        store=True,
+        readonly=False,
+        index=True,
+        tracking=True,
     )
     equipment_ids = fields.Many2many("fsm.equipment")
 
+    @api.depends("location_id")
+    def _compute_person_id(self):
+        """Compute the person from the location's territory"""
```

Comment:
If I recall correctly, you may not use territory and assing person directly

Reply:
The territory comes from the location (related to `location_id.territory_id`)
So the goal here is to populate the order's assigned user from the related territory's partner (if set)

Reply:
Yes but leaving person_id  blank is a feature.

The use of territory_id on location is optionnal.

I have business case where the vendor create the FSM Order and set a location (and leave person_id with no value), then (few days after) some manager assign a person to the fsm order.

Reply:
That case still works, you just don't set any territory on the location, or no Assigned user on the territory.
Isn't it enough for your use case?

Reply:
If you need a "default_person_id" on territory or fsm_location, why not just create one ?
It will be more explicit about what it does.